starters/solidjs-tanstack-tailwind/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.jsx]
+quote_type = single
```

Comment:
Does this work? It doesn't seem to be in the [editorconfig spec](https://editorconfig-specification.readthedocs.io/) nor in the [Prettier API](https://prettier.io/docs/en/api.html)

Reply:
I think at the moment it's a proposal.
So, should we take it out?
@mbicknese 

![Screenshot 2022-10-18 at 17 46 58](https://user-images.githubusercontent.com/28502531/196492993-be7ed3c6-591d-46d2-9456-072afcb6ba71.jpg)

Reply:
That's pretty cool, let's roll with it. 👍

Reply:
Here is the link for the screenshot.
https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties